app/assets/javascripts/backbone/views/page_view.coffee

```diff
@@ -0,0 +1,12 @@
+class MS.Views.PageView extends Backbone.View
+
+  el: 'body'
+
+  events: (params) ->
+    'change select[name="division_id"]': 'selectDivision'
+
+  selectDivision: (e) ->
+    $item = $(e.currentTarget)
+    $("input[name=redisplay_url]").val(window.location)
```

Comment:
Isn't this bound to lead to 404s errors, if, e.g., the user is on a specific loan page and then changes divisions?